### PR TITLE
Disc

### PR DESCRIPTION
**Pull request by DawidSzpener**

### CODE REVIEW

File: src/components/App/App.jsx [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
 import Board from '../Board/Board'
+import { Text } from '../Text/Text'
+import './App.css'
 
 const App = () => (
+  
   <div className="app-container">
+    <div className="description-div">
+      <Text />
+    </div>
```
Comment (ffgi-es) [inline src/components/App/App.jsx]: I think this div should be held in the Text component itself.

File: src/components/App/App.spec.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ describe('App', () => {
   })
 
   it('should render a <div />', () => {
-    expect(wrapper.find('div').length).toEqual(1)
+    expect(wrapper.find('div')).toBeDefined()
```
Comment (ffgi-es) [inline src/components/App/App.spec.js]: This shouldn't need to be changed if the div for text-description is moved into the div component

File: src/components/Text/Text.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+import logo from './logo.png'
+import './Text.css'
+
+export const Text = () => <Fragment>
+  <img src={logo} alt="Logo" width="400" height="200" />
+  <br></br><br></br><br></br>
+  Is a collection of cells arranged in a grid,<br></br>
+  such that each cell changes state as a function of time according to
+  a defined set of rules that includes the states of neighboring cells.
+</Fragment>
```
Comment (ffgi-es) [inline src/components/Text/Text.jsx]: This should be wrapped in the div I've been talking about

Comment (DawidSzpener) [reply src/components/Text/Text.jsx]: Yup, done.

**Pull request merged**